README.md

```diff
@@ -209,10 +208,10 @@ To learn more, see:
     exports.handler = signalFxLambda.asyncWrapper(async (event, context) => {
       ...
       // to send custom event:
-      await signalFxLambda.helper.sendCustomEvent('Custom', {functionName: context.functionName}, {description: 'Custom event'});
+      signalFxLambda.helper.sendCustomEvent('Custom', {functionName: context.functionName}, {description: 'Custom event'});
             
       // to transform & forward CloudWatch event:
-      await signalFxLambda.helper.sendCloudWatchEvent(event);
+      signalFxLambda.helper.sendCloudWatchEvent(event);
```

Comment:
same comment as above. If we don't wait for the function to finish exporting, there is no guarantee the data will actually be reported as function may exit and execution may be stopped before the metrics lib has a chance to complete the export.